EU4toV2/Source/V2World/State/StateNameParser.cpp

```diff
@@ -0,0 +1,75 @@
+#include "StateNameParser.h"
+#include "Configuration.h"
+#include "Log.h"
+#include "OSCompatibilityLayer.h"
+#include <fstream>
+#include <filesystem>
+namespace fs = std::filesystem;
+
+V2::StateNameParser::StateNameParser()
+{
+	if (commonItems::DoesFileExist("./blankMod/output/localisation/text.csv"))
+	{
+		importStateLocalizations("./blankMod/output/localisation/text.csv");
+	}
+	const auto& stateLocPath = theConfiguration.getVic2Path() + "/localisation";
```

Comment:
ditto as above, you probably want HPM locs, not vanilla ones., to overwrite our locs, if and only if HPM is enabled.